### PR TITLE
changes from August21 review

### PR DESCRIPTION
**Pull request by DennisNeuendorfJimdo**
… Adopt

Read the Changelog [here](https://jimplan.atlassian.net/wiki/spaces/ENGINEERING/pages/2816999428/Tech+Radar+Change+Log#[hardBreak]Review-44th-of-August[hardBreak])

### CODE REVIEW

File: data/frontend.csv [hunk 1]
```diff
@@ -5,8 +5,10 @@ Loki,Adopt,CI Tools,FALSE,
 Prettier,Adopt,CI Tools,FALSE,
 ESLint,Adopt,CI Tools,FALSE,
 Stylelint,Adopt,CI Tools,FALSE,
-Test Cafe,Trial,CI Tools,FALSE,
-Cypress,Assess,CI Tools,TRUE,
+Test Cafe,Adopt,CI Tools,FALSE,
+ ,Trial,CI Tools,FALSE,
+ ,Assess,CI Tools,TRUE,
```
Comment (jitendra-jimdo) [inline data/frontend.csv:10]: These 2 lines are for? 
Looks like we don't need them, right?

**Comment by jitendra-jimdo** [comment]
@DennisNeuendorfJimdo We discussed briefly about documenting why we are doing these changes in the tech radar, Did you plan to add some notes in the PR, or you planned something else?

**Review by jitendra-jimdo — APPROVED** [review]

File: data/frontend.csv (hunk 1 above)
Comment (mathiswiehl) [reply data/frontend.csv:10]: What was the reason for leaving those in?

Comment (jitendra-jimdo) [reply data/frontend.csv:10]: I think it was to show these empty entries for Trial and Assess
Screenshot 2021-10-01 at 08 20 32

Comment (mathiswiehl) [reply data/frontend.csv:10]: ok makes sense ✅ Thanks 🙏

**Pull request merged**